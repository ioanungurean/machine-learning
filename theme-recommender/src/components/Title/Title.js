import React, { Component } from 'react';
import PropTypes from 'prop-types';

import './Title.scss';

class Title extends Component {
  render() {
    return (
      <span className="title">
        {this.props.title}
      </span>
    );
  };
};

Title.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Title;
