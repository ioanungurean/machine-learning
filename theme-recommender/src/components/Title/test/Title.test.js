import React from 'react';
import { shallow } from 'enzyme';

import Title from '../Title';

describe('<Title />', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<Title title="Test Title" />);

    expect(wrapper).toMatchSnapshot();
  });
});
