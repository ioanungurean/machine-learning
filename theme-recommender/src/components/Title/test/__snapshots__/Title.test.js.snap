// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render without throwing an error 1`] = `
<span
  className="title"
>
  Test Title
</span>
`;
